src/components/Popover/Options.md

```diff
@@ -0,0 +1,90 @@
+```jsx
+import { useState } from 'react'
+import { Button, Spacer, TitleSmall2 } from '../index'
+import useScrollItemIntoView from '../../hooks/useScrollItemIntoView'
+import usePopoverNavigation from '../../hooks/usePopoverNavigation'
+import styles from './Example.module.scss'
+
+const ExamplePopoverConsumer = () => {
+  const [isOpen, setIsOpen] = useState(false)
+  const [activeOption, setActiveOption] = useState(0)
+  const [selectedOption, setSelectedOption] = useState(-1)
+  const { scrollItemIntoView } = useScrollItemIntoView()
+  const { handlePopoverNavigation } = usePopoverNavigation()
+
+  // Whatever you actually care about from popover options
+  const [location, setLocation] = useState('')
+  const [lastSelected, setLastSelected] = useState('Select something')
+
+  const items = [
+    {
+      name: 'Macedonia',
+    },
+    {
+      name: 'Madagascar',
+    },
+    {
+      name: 'Maldives',
+    },
+    {
+      name: 'Mexico',
+    },
+  ]
+
+  const optionsRefs = Array(items.length)
+    .fill(0)
+    .map(() => React.createRef())
+
+  const handleOnChange = (item) => {
+    setLastSelected(item.name)
+  }
+
+  const setSelectedAndActiveOptions = (index) => {
+    setSelectedOption(index)
```

Comment:
The difference between 'active' and 'selected' is not very clear when reading this and the other files.  Maybe it can be made clearer - either by renaming one of them to be more specific (active == ?) or updating the readme with some explanation?

Reply:
Is 'active' == 'highlighted'?

Reply:
I think you're questions and the confusion totally make sense. Yes to the second question, it's the "highlighted" visually, but really it's the focused item since these are used to support the a11y keyboard navigability.

It's my Sunday and I have my kids, but I did just add a commit which I think best addresses this as I hadn't documented these props!

```js
Option.propTypes = {
  /** The currently active option item. This is the item that is currently
   * being focused on (but not necessarily selected yet!) */
  currentActive: PropTypes.number,
  /** The option item's index in the options */
  itemIndex: PropTypes.number,
  /** The data object with the data for this option via the `dataKey` */
  item: PropTypes.object,
  /** The key to access via `item` e.g. `item.name` */
  dataKey: PropTypes.string,
  /** The currently selected option item. Unlike an active item, this signifies
      that the user has actually selected or chosen this item. Of course an
      item can be both active and selected. */
  selectedOption: PropTypes.number,
  /** Callback that gets called once an option is selected */
  setSelectedAndActiveOptions: PropTypes.func,
  /** Callback that gets called on change */
  onChange: PropTypes.func,
}
```

I still like active better then highlighted, so I'd like to keep the variable names as is with hopes that improved docs takes care of this being unclear. Thanks for the feedback Sriram!

Reply:
I also think 'active' is better and I think your comment helps in understanding this.